.github/workflows/deploy.yml

```diff
@@ -0,0 +1,40 @@
+name: Build, push and deploy on push to master
+
+on:
+    push:
+        branches:
+            - master
+
+jobs:
+    deploy:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
```

Comment:
As someone not terribly well versed in this: What is the checkout@v2?

Reply:
This is simply version 2 of a github action that is used to "checkout" the code (pull it). The cool thing about github actions is that they correspond to a repository. That means that every time you're in doubt what a action does, you can simply look at the repository corresponding to the action (example: actions/checkout -> https://github.com/actions/checkout).